src/javascript/tracking.js

```diff
@@ -4,10 +4,10 @@ import {init as initSession} from './core/session.js';
 import {init as initSend} from './core/send.js';
 import {event} from './events/custom.js';
 import {page} from './events/page-view.js';
-import {init as initClick} from './events/click.js';
+import {click} from './events/click.js';
```

Comment:
Thanks for looking into this @notlee 🙂 It looks like `'./events/click.js'` only has one named export `init`, so I'm not sure this doing what is expected 🤔